examples/lock-app/cc13x4_26x4/chip.syscfg

```diff
@@ -191,15 +191,16 @@ ble.numOfAdvSets                                          = 1;
 ble.lockProject                                           = true;
 ble.oneLibSizeOpt                                         = true;
 ble.maxPDUSize                                            = 255;
-ble.radioConfig.codeExportConfig.$name                    = "ti_devices_radioconfig_code_export_param1";
+ble.radioConfig.codeExportConfig.$name                    = "ti_devices_radioconfig_code_export_param0";
```

Comment:
This change is required to avoid the following build error. 
```
error: /ti/ble5stack/ble radioConfig.codeExportConfig.$name: Duplicate name: 'ti_devices_radioconfig_code_export_param1' also exists on instance(s) of Code Export Configuration
error: /ti/devices/radioconfig/custom radioConfigieee154.codeExportConfig.$name: Duplicate name: 'ti_devices_radioconfig_code_export_param1' also exists on instance(s) of Code Export Configuration
```

The build for  `LP_EM_CC1454P10_6` only worked as something in the redesign of syscfg renamed the `ble.radioConfig.codeExportConfig.$name ` when setting `ble.rfDesign` to `LP_EM_CC1354P10_6`. 

`ti_devices_radioconfig_code_export_param0` seem a more correct name as the ble codeExportConfig is set to this for other ble examples.


Below was used to debug and confirm this.

```
+console.log("RFCustom.radioConfigieee154p10.codeExportConfig: ", ble.radioConfig.codeExportConfig.$name);
 /* BLE */
 ble.addressMode                                           = "ADDRMODE_RP_WITH_PUBLIC_ID";
 ble.maxConnNum                                            = 1;
@@ -194,6 +195,7 @@ ble.lockProject                                           = true;
 ble.oneLibSizeOpt                                         = true;
 ble.maxPDUSize                                            = 255;
 ble.radioConfig.codeExportConfig.$name                    = "ti_devices_radioconfig_code_export_param1";
+console.log("ble.radioConfig.codeExportConfig.$name         : ", ble.radioConfig.codeExportConfig.$name);
 ble.connUpdateParamsPeripheral.$name                      = "ti_ble5stack_general_ble_conn_update_params0";
 ble.connUpdateParamsPeripheral.reqMinConnInt               = 30;
 ble.connUpdateParamsPeripheral.reqMaxConnInt               = 50;
@@ -202,6 +204,8 @@ ble.advSet1.$name                                         = "ti_ble5stack_broadc
 ble.advSet1.advParam1.$name                               = "ti_ble5stack_broadcaster_advertisement_params0";
 
 ble.rfDesign                                 = "LP_EM_CC1354P10_6";
+console.log("ble.radioConfig.codeExportConfig.$name         : ", ble.radioConfig.codeExportConfig.$name)
+error()


RFCustom.radioConfigieee154p10.codeExportConfig:  ti_devices_radioconfig_code_export_param0
ble.radioConfig.codeExportConfig.$name         :  ti_devices_radioconfig_code_export_param1
ble.radioConfig.codeExportConfig.$name         :  ti_devices_radioconfig_code_export_param0
```

Generating an output of:

```
RFCustom.radioConfigieee154p10.codeExportConfig:  ti_devices_radioconfig_code_export_param0
ble.radioConfig.codeExportConfig.$name         :  ti_devices_radioconfig_code_export_param1
ble.radioConfig.codeExportConfig.$name         :  ti_devices_radioconfig_code_export_param0
```